test/unittest/symbolic-node.jl

```diff
@@ -100,6 +101,7 @@ function test_attrs()
   @test isnull(mx.get_attr(conv, :b))
   @test get(mx.get_attr(conv, :a)) == "a"
   @test get(mx.get_attr(conv, :π)) == "π"
+  @test isa(mx.get_name(conv), Symbol)
```

Comment:
Could you add a test for a grouped symbol to reflect the behavior you specified in this situation? Thanks!